aiocouch/database.py

```diff
@@ -64,6 +64,8 @@ def __init__(self, couchdb, id):
         await self._delete()
 
     async def docs(self, ids=None, create=False, prefix=None, **params):
+        if ids is not None and not ids:
```

Comment:
May I suggest:
```suggestion
        if ids is not None and len(ids) == 0:
```
? 

I think it's a bit clearer for developers.

Reply:
Thanks for the feedback @adrienverge, I updated the commit accordingly.